contrib/curl-cmake/CMakeLists.txt

```diff
@@ -153,6 +153,22 @@ set (SRCS
 
 add_library (_curl ${SRCS})
 
+set(CURL_LIBS "")
+if(APPLE)
+
+    find_library(SYSTEMCONFIGURATION_FRAMEWORK "SystemConfiguration")
```

Comment:
We don't allow `find_library` in our cmake files.

Reply:
Hi @alexey-milovidov, any specific reason we are restricting the find_library in cmake files?

Reply:
It is needed for hermetic builds - to avoid dependence on the environment.

Reply:
If the new curl version started to need something from SystemConfiguration,
maybe better to chop it from the library?